pkg/oc/cli/util/clientcmd/factory_client_access.go

```diff
@@ -63,7 +62,7 @@ func NewClientAccessFactory(optionalClientConfig kclientcmd.ClientConfig) Client
 	clientConfig := optionalClientConfig
 	if optionalClientConfig == nil {
 		// TODO: there should be two client configs, one for OpenShift, and one for Kubernetes
-		clientConfig = osclientcmd.DefaultClientConfig(flags)
```

Comment:
the dependency in this direction is ok.  You have a lot of them and oc has to vendor origin anyway.